badgr-fullstack-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,282 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "defaultValue": "badgruser",
+      "metadata": {
+        "description": "Administrator username."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Administrator password."
+      }
+    },
+	"badgrSuperUserEmail": {
+      "type": "string",
+      "metadata": {
+        "description": "Badgr Application super user password."
+      }
+    },
+	"badgrSuperUsername": {
+      "type": "string",
+      "defaultValue": "badgruser",
+      "metadata": {
+        "description": "Badgr Application super username."
+      }
+    },
+    "badgrSuperUserPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Badgr Application super user password."
+      }
+    },
+    "dnsLabelPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "DNS Label for the Public IP. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+      }
+    },
+    "vmSize": {
+      "type": "string",
+      "defaultValue": "Standard_D3_v2",
+      "allowedValues": [
+        "Standard_A2",
+        "Standard_A3",
+        "Standard_A4",
+        "Standard_A5",
+        "Standard_A6",
+        "Standard_A7",
+        "Standard_A8",
+        "Standard_A9",
+        "Standard_A10",
+        "Standard_A11",
+        "Standard_D2",
+        "Standard_D3",
+        "Standard_D4",
+        "Standard_D11",
+        "Standard_D12",
+        "Standard_D13",
+        "Standard_D14",
+        "Standard_D2_v2",
+        "Standard_D3_v2",
+        "Standard_D4_v2",
+        "Standard_D5_v2",
+        "Standard_D11_v2",
+        "Standard_D12_v2",
+        "Standard_D13_v2",
+        "Standard_D14_v2",
+        "Standard_D15_v2"
+      ],
+      "metadata": {
+        "description": "Virtual machine size."
+      }
+    }
+  },
+  "variables": {
+    "apiVersion": "2015-06-15",
+    "scriptDownloadUri": "https://raw.githubusercontent.com/satyarapelly/azure-quickstart-templates/master/badgr-fullstack-ubuntu/",
+    "installScript": "install-badgr.sh",
+    "installCommand": "[concat('bash -c ''nohup ./', variables('installScript'), ' -u ', parameters('badgrSuperUsername'), ' -p ', parameters('badgrSuperUserPassword'), ' -e ', parameters('badgrSuperUserEmail'),' </dev/null &>/var/log/azure/badgr-install.log &''')]",
+    "vmName": "badgrvm",
```

Comment:
Yes, it is easy to read and maintainable,  easy to locate.
thanks for the comment.